src/main/java/org/jenkinsci/plugins/github_branch_source/Connector.java

```diff
@@ -139,9 +156,9 @@ public static FormValidation checkScanCredentials(SCMSourceOwner context, String
     }
 
     @CheckForNull
-    public static StandardCredentials lookupScanCredentials(@CheckForNull SCMSourceOwner context,
-                                                                          @CheckForNull String apiUri,
-                                                                          @CheckForNull String scanCredentialsId) {
+    public static StandardCredentials lookupScanCredentials(@CheckForNull Item context,
```

Comment:
I'd rather merge this first as that way I can be sure that we have not leaked a required connection check